vnext/ReactUWP/Views/SIPEventHandler.cpp

```diff
@@ -20,38 +21,79 @@ namespace react {
 namespace uwp {
 
 SIPEventHandler::SIPEventHandler(const std::weak_ptr<IReactInstance> &reactInstance)
-    : m_wkReactInstance(reactInstance) {
-  auto coreInputView = winrt::CoreInputView::GetForCurrentView();
-  if (coreInputView) {
-    m_occlusionsChanged_revoker = coreInputView.OcclusionsChanged(
-        winrt::auto_revoke, [=](auto &&, const winrt::CoreInputViewOcclusionsChangedEventArgs &e) {
+    : m_wkReactInstance(reactInstance), m_fireKeyboradEvents(false){};
+
+SIPEventHandler::~SIPEventHandler() {
+  m_occlusionsChanged_revoker = {};
+}
+
+void SIPEventHandler::AttachView(XamlView xamlView, bool fireKeyboardEvents) {
+  m_fireKeyboradEvents = fireKeyboardEvents;
+
+  if (Is19H1OrHigher()) {
```

Comment:
>Is19H1OrHigher [](start = 6, length = 14)

Suggest you use if (auto uie = try_as<winrt::IUIElement10>().
It's possible we don't need to make code change in the future when integrate with winui3.

Reply:
It is there mainly for CoreInputView::GetForUIContext, last heard there is no plan to bring that down level yet.. We can always reevaluate at later time by searching the code where we call the platform check such as Is19H1orHigher..

---
In reply to: [347710901](https://github.com/microsoft/react-native-windows/pull/3665#discussion_r347710901) [](ancestors = 347710901)

Reply:
Actually I realized CoreInputView.TryShow/TryHide is not availiable until RS5, so more platform check is needed. And I probably need to fall back to InputPane API on earlier OS.

---
In reply to: [348209889](https://github.com/microsoft/react-native-windows/pull/3665#discussion_r348209889) [](ancestors = 348209889,347710901)